app/book.py

```diff
@@ -0,0 +1,22 @@
+class Book:
+    def __init__(self, title: str, content: str) -> None:
+        self.title = title
+        self.content = content
+
+    def display(self, display_type: str) -> None:
+        if display_type == "console":
+            print(self.content)
+        elif display_type == "reverse":
```

Comment:
think how to separate the responsibilities of displaying and printing into distinct classes